base/system/diskpart/lang/pl-PL.rc

```diff
@@ -49,9 +49,9 @@ BEGIN
     IDS_LIST_PARTITION_LINE "  -------------  ----------------  -------  ------------\n"
     IDS_LIST_PARTITION_FORMAT "%c Partycja    %2lu   %-16s  %4I64u %-2s  %4I64u %-2s\n"
     IDS_LIST_PARTITION_NO_DISK "\nNie wybrano dysku do wyświetlenia partycji.\nWybierz dysk i spróbuj ponownie.\n\n"
-    IDS_LIST_VOLUME_HEAD "  Volume ###  Ltr  Label        FS     Type        Size     Status   Info\n"
-    IDS_LIST_VOLUME_LINE "  ----------  ---  -----------  -----  ----------  -------  -------  --------\n"
-    IDS_LIST_VOLUME_FORMAT "  Volume %3lu   %c   %-11s  %-5s  %10u  %4I64u %-2s\n"
+    IDS_LIST_VOLUME_HEAD "  Wolumin ###  Lit  Etykieta     FS     Typ         Rozmiar  Stan     Info\n"
+    IDS_LIST_VOLUME_LINE "  -----------  ---  -----------  -----  ----------  -------  -------  --------\n"
+    IDS_LIST_VOLUME_FORMAT "  Wolumin %3lu   %c   %-11s  %-5s  %10u  %4I64u %-2s\n"
```

Comment:
please attach a picture demonstrating that the indentation of this table does still look like intended after your changes.

Reply:
You're funnny, you know?�
![reactos_diskpart_volume](https://user-images.githubusercontent.com/40692062/161370225-866c9342-4d93-4f79-8341-6b5ad2e00486.png)
�

Reply:
The "indentation ", should/must be bigger lok on winnt 5.2
![winnt5x_diskpart_volume](https://user-images.githubusercontent.com/40692062/161370824-8c02875a-596a-4a2a-adc9-24f3f929399b.png)

Reply:
I see. Please file a fresh ticket about that!